Workbooks/ADXCluster/AtResourceCache/ResourceCacheDetails.workbook

```diff
@@ -238,7 +238,7 @@
               "title": "Queries look back period (X-axis) and the number of queries (Y-axis)",
               "noDataMessage": "There were no queries on this table",
               "timeContext": {
-                "durationMs": 0
+                "durationMs": 604800000
```

Comment:
What is this for?

Reply:
It automatically changes. It does not affect on the UI or the timespan.